README.md

```diff
@@ -23,6 +23,7 @@ For other linux distributions, the daemon binary needs to be built from source c
 
 - Active Directory server ( Windows Server )
 - Linux instances or hosts that are domain-joined to Active Directory
+  - Unless you are using Amazon Elastic Container Service - Fargate. In that case you must [use domainless gMSA](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/fargate-linux-gmsa.html#fargate-linux-gmsa-considerations).
```

Comment:
This is true, but I think we should also include the fact that you can use Domainless with ECS without Fargate as described here: https://aws.amazon.com/about-aws/whats-new/2023/07/amazon-ecs-domainless-gmsa-authentication-linux-windows-containers/

+ @as14692 or @jamolina who would be able to best word it.

Reply:
@SovietFrontier gMSA for EC2 Linux containers on Amazon ECS provides the option of domainless gMSA and joining each instance to a single domain. Please update to reflect it as follows

- [EC2 Linux containers on Amazon ECS](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/linux-gmsa.html#linux-gmsa-considerations) provides the option of domainless gMSA and joining each instance to a single domain
- [Linux containers on Fargate](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/fargate-linux-gmsa.html#fargate-linux-gmsa-considerations) must use domainless gMSA

Reply:
@SovietFrontier gMSA for EC2 Linux containers on Amazon ECS also provides the option of domainless gMSA along with the option of joining each instance to a single domain. Please update to reflect it as follows
```suggestion
  - [EC2 Linux containers on Amazon ECS](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/linux-gmsa.html#linux-gmsa-considerations) provides the option of domainless gMSA and joining each instance to a single domain
  - [Linux containers on Fargate](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/fargate-linux-gmsa.html#fargate-linux-gmsa-considerations) must use domainless gMSA
```

Reply:
Done! Thank you for that wording.